src/CommonITILObject.php

```diff
@@ -3759,7 +3759,7 @@ public static function dropdownStatus(array $options = [])
 
         switch ($p['showtype']) {
             case 'allowed':
-                $tab = static::getAllowedStatusArray($p['value']);
+                $tab = static::getAllowedStatusArray($p['value_calculation'] ?? $p['value']);
```

Comment:
Maybe change to:
``$tab = static::getAllowedStatusArray(empty($p['value_calculation'])?$p['value']:$p['value_calculation']);``
or update template is now passing also empty value during the item creation when fields['status'] does not exist:
'value_calculation': item.fields['status']